Homework #1&2

eclimov opened the pull request:
Implemented requirements

eclimov committed:
- initialized yarn project
- added webpack
- added vue
- added autoprefixer
- moved webpack to devDependencies
- partially set up webpack

eclimov committed:
- Added required packages

eclimov committed:
- Fixed lint warnings

eclimov committed:
- Removed unnecessary config

eclimov committed:
- Added webpack dev config
- Updated webpack config
- Updated build config
- Added images

eclimov committed:
- Added an empty line at the end of each file according to Github recommendations

rapi commented on package.json:
```diff
@@ -0,0 +1,45 @@
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "dependencies": {
```
think about dev and prod dependence

eclimov replied:
Done

eclimov committed:
- Moved some dependencies to devDependencies

eclimov commented:
Done

eclimov committed:
- Implemented functions:
get first 30 Fibonacci numbers using recursion;
count sum of numbers using currying with closing brackets;
count sum of numbers using currying without closing brackets;
- Slightly updated eslint config to avoid annoying warning

eclimov commented:
**Completed the 2nd homework:**
1. Implemented the three functions
2. Achieved level 6 on Codewars:
https://www.codewars.com/users/eclimov
